dashdotdb/schemas/swagger.yaml

```diff
@@ -209,6 +209,11 @@ paths:
     get:
       summary: Returns value and target for queried Service SLO Metrics
       parameters:
+      - name: slo_doc_name
+        in: query
+        required: true
```

Comment:
This is a relatively strong API change. Do we control all dashdotdb API clients ?

Reply:
An excellent point. I will rethink the `required` property here.